app/data/2_0001.json

```diff
@@ -0,0 +1,109 @@
+                                            "q_code": "53",
+                                            "type": "PositiveInteger"
+                                        },
+                                        {
+                                            "id": "number-of-employees-female-less-30-hours",
+                                            "label": "Number of female employees working 30 hours or less per week",
+                                            "mandatory": false,
+                                            "q_code": "54",
+                                            "type": "PositiveInteger"
+                                        },
+                                        {
+                                            "id": "number-of-employees-total",
+                                            "label": "Total number of employees",
+                                            "mandatory": true,
+                                            "q_code": "50",
+                                            "type": "PositiveInteger"
+                                        }
+                                    ],
+                                    "guidance": [
+                                        {
+                                            "list": [
+                                                "all workers paid directly from {{respondent.ru_name}}’s payroll(s)",
+                                                "those temporarily absent but still being paid, for example on maternity leave"
+                                            ],
+                                            "title": "Include:"
+                                        },
+                                        {
+                                            "list": [
+                                                "agency workers paid directly from the agency payroll",
+                                                "voluntary workers",
+                                                "former employees only receiving a pension",
+                                                "self-employed workers",
+                                                "working owners who are not paid via PAYE"
+                                            ],
+                                            "title": "Exclude:"
+                                        }
+                                    ],
+                                    "id": "number-of-employees-question",
+                                    "title": "On {{exercise.start_date|format_date}}  what was the number of employees for {{respondent.ru_name}}?",
+                                    "type": "General"
+                                }
+                            ],
+                            "title": "Quarterly Business Survey"
+                        }
+                    ],
+                    "type": "Questionnaire"
```

Comment:
Are we missing the description

An employee is anyone aged 16 years or over that your organisation directly pays from its payroll(s), in return for carrying out a full-time or part-time job or being on a training scheme.